### PR TITLE
{math}[foss/2023b] PaStiX v6.3.2

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PaStiX/PaStiX-6.3.2-foss-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://files.inria.fr/pastix/releases/v6/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['c4da8802d1933eecf8c09d7e63c014c81ccf353fe623142e9f5c5fc65ed82ee0']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+]
```
Comment (branfosj) [inline easybuild/easyconfigs/p/PaStiX/PaStiX-6.3.2-foss-2023b.eb:19]:
```suggestion
    ('pkgconf', '2.0.3'),
]
```

```
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.4.2") 
```

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen3

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/41de74f3ccbc0a9774b2ee0dede10f55 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen3

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @SimonPinches!

**Comment by SimonPinches** [comment]
Thanks @branfosj !

**Pull request merged**